modules/Retry.py

```diff
@@ -0,0 +1,33 @@
+from logging import debug, error, info
+from time import sleep
+from ast import literal_eval
+
+class DebomaticModule_Retry:
+    def __init__(self):
+        pass
+
+    def post_build(self, args):
+        if args.success:
+            return
+
+        map = dict()
+        if args.opts.has_section('retry'):
+            map = literal_eval(args.opts.get('retry', 'dists'))
+
+        failed_dist = args.distribution
+        if not failed_dist in map:
+            debug('not requeueing %s failure (%s not in retry.dists)' % (args.package, failed_dist))
+            return
+
+        retry_dist = map[failed_dist]
+        info('requeuing %s in %s' % (args.package, retry_dist))
+
+        incoming = args.opts.get('debomatic', 'incoming')
+        commands_file = 'retry-%s-%s-%s.commands' % (failed_dist, retry_dist, args.package)
+
+        with open('%s/%s' % (incoming, commands_file), 'w') as fd:
+            command = 'rebuild %s %s' % (args.package, retry_dist)
+            fd.write(command + '\n')
```

Comment:
I haven't checked yet, but I think providing an unsigned command file would cause its rejection if GPG support is enabled.

Reply:
Indeed. Obviously we don't have gpg support enabled. Do you have any suggestions on how to tackle this? I suppose it could sign the file if it has an encryption key in the keyring.